Source/WebCore/Modules/speech/SpeechSynthesis.cpp

```diff
@@ -243,42 +242,42 @@ void SpeechSynthesis::didStartSpeaking()
 {
     if (!m_currentSpeechUtterance)
         return;
-    didStartSpeaking(*m_currentSpeechUtterance->platformUtterance());
+    didStartSpeaking(*m_currentSpeechUtterance->utterance()->platformUtterance());
```

Comment:
We should add another member function named like protectedUtterance() which returns RefPtr and use that here instead.